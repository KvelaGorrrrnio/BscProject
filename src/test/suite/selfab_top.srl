list list list int N

init N [4,4,4]
N[2,3,3] += top (top N[2])
