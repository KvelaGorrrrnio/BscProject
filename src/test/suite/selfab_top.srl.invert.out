list  list  list  int N

N[2,0,0] -= ^ (^ N[2])
free N [4,4,4]
